data-tck/src/main/java/io/micronaut/data/tck/entities/ContactView.java

```diff
@@ -12,6 +12,7 @@
 public class ContactView {
     @Id
     @GeneratedValue(GeneratedValue.Type.IDENTITY)
+    @JsonProperty("_id")
```

Comment:
For some reason, `MappedEntityVisitor` is not processing this class and doesn't add wanted annotations. There are 
```
...
testImplementation projects.micronautDataTck
...
testAnnotationProcessor projects.micronautDataProcessor
...
```
but not sure why this class in data-tck module is not being visited in the data-r2dbc tests.